.gitignore

```diff
@@ -14,3 +14,4 @@ test_data
 __debug_bin*
 llama/build
 llama/vendor
+vendor
```

Comment:
`vendor` should already be ignored by the line above. If that's not the case, please provide more details

Reply:
`llama/vendor` only ignore vendor in llama directory

run `go mod vendor` and git status:
```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	vendor/

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
Ollama doesn't vendor go modules so this is inapplicable

Reply:
Running `go mod vendor` in the root directory of the project is a very common behavior, I just think adding it to gitignore can keep the local git clean, of course, this is not a bug.

Reply:
I'm not making a comment on vendoring go modules in general, I'm saying Ollama does not vendor go modules. As such, it's inapplicable as a gitignore entry for this repo. You're welcome, of course, to vendor it in your clone/fork and make this change there